megamek/src/megamek/common/Entity.java

```diff
@@ -217,6 +217,9 @@
     protected boolean useManualBV = false;
     protected int manualBV = -1;
 
+    protected int initialBV = -1;
+    protected boolean initialBVSet = false;
```

Comment:
I think I would agree with you.  My initial thought was that we would want to make sure this value doesn't get updated more than once.  But, after submitting the pull request, I came to the conclusion that it was superfluous, so I'll remove it.